### DISCUSSION TITLE
Hello, why is there no quartz.simple.remotingschedulerexporter in my 3.4 version

### DISCUSSION THREAD

**Discussion by lshsuper** [quartznet/quartznet#1739] [Q&A]

**Comment by lshsuper** [comment]
![1662870021964](https://user-images.githubusercontent.com/31640109/189512263-54bcf91b-1a8d-4632-b7ab-adc394d8ca02.png)

**Comment by lahma** [comment]
NETStandard doesn't contain remoting infrastructure, remoting exporter is part of full framework part of the library so you need to target the full framework (non-core/NET5/NET6).

**Reply by lshsuper** [reply]
Is my project that references quartz.net (net5)? I have changed it to, but there is still no such class

![45d1115d866a0ed31553a9fc926feb6](https://user-images.githubusercontent.com/31640109/189527637-85f67a80-bfae-4e82-831e-c437e8a77d1a.png)

![e70bb840f9497b1562cddad195930ab](https://user-images.githubusercontent.com/31640109/189527641-992296c0-afb8-4d31-8a32-a472aa71d226.jpg)

**Reply by lahma** [reply]
You need to reference net4.6.2-net481. New core does not have remoting.

**Reply by lshsuper** [reply]
In the net core project, how can the scheduler that controls other service nodes be called remotely?

**Reply by lahma** [reply]
Nothing built-in at the moment, see https://github.com/quartznet/quartznet/issues/511
